settings/development/PandoraSettings_Streaming_Neutrino_DUNEFD.xml

```diff
@@ -0,0 +1,640 @@
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewClearTracks">
+                <MinMatchingScore>0.0</MinMatchingScore>
+                <MinLocallyMatchedFraction>0.30</MinLocallyMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewLongTracks">
+                <MinMatchingScore>0.0</MinMatchingScore>
+                <MinMatchedFraction>0.30</MinMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewLongTracks">
+                <MinMatchingScore>0.95</MinMatchingScore>
+                <MinMatchedFraction>0.0</MinMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewSimpleTracks">
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+        </TrackTools>
+    </algorithm>
+    <!--2 view Calo matching for showers - V W -->
+    <algorithm type = "LArTwoViewTransverseTracks">
+        <InputClusterListName1>ShowerClustersV</InputClusterListName1>
+        <InputClusterListName2>ShowerClustersW</InputClusterListName2>
+        <OutputPfoListName>ShowerParticles3D</OutputPfoListName>
+        <LocalMatchingScoreThreshold>0.90</LocalMatchingScoreThreshold>
+        <DisplayPotentialMatches>false</DisplayPotentialMatches>
+        <MinSamples>13</MinSamples>
+        <TrackTools>
+            <tool type = "LArTwoViewThreeDKink"/>
+            <tool type = "LArTwoViewClearTracks">
+                <MinMatchingScore>0.95</MinMatchingScore>
+                <MinLocallyMatchedFraction>0.0</MinLocallyMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewClearTracks">
+                <MinMatchingScore>0.0</MinMatchingScore>
+                <MinLocallyMatchedFraction>0.30</MinLocallyMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewLongTracks">
+                <MinMatchingScore>0.0</MinMatchingScore>
+                <MinMatchedFraction>0.30</MinMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewLongTracks">
+                <MinMatchingScore>0.95</MinMatchingScore>
+                <MinMatchedFraction>0.0</MinMatchedFraction>
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+            <tool type = "LArTwoViewSimpleTracks">
+                <PrintIfMatch>false</PrintIfMatch>
+            </tool>
+        </TrackTools>
+    </algorithm>
+    <!-- end two view recovery -->
+
+    <!-- ThreeDRecoveryAlgorithms -->
+    <algorithm type = "LArVertexBasedPfoRecovery">
+        <InputClusterListNames>TrackClustersU TrackClustersV TrackClustersW ShowerClustersU ShowerClustersV ShowerClustersW</InputClusterListNames>
```

Comment:
Was wondering about this small block, whether everything made should be called a track, or confine to the streams? Is this first place that streams cross for Pfo formation?

Reply:
Difficult to know how best to deal with this one, but the vertex based recovery doesn't work if it can't see all clusters.